Source/WebCore/rendering/svg/RenderSVGResourceClipper.cpp

```diff
@@ -191,6 +191,24 @@ void RenderSVGResourceClipper::applyMaskClipping(PaintInfo& paintInfo, const Ren
     frameView.setPaintBehavior(oldBehavior);
 }
 
+FloatRect RenderSVGResourceClipper::calculateClipContentRepaintRect(RepaintRectCalculation repaintRectCalculation)
+{
+    FloatRect clipContentRepaintRect;
+    // This is a rough heuristic to appraise the clip size and doesn't consider clip on clip.
+    for (Node* childNode = clipPathElement().firstChild(); childNode; childNode = childNode->nextSibling()) {
+        RenderObject* renderer = childNode->renderer();
+        if (!childNode->isSVGElement() || !renderer)
+            continue;
+        if (!renderer->isRenderOrLegacyRenderSVGShape() && !renderer->isRenderSVGText() && !childNode->hasTagName(SVGNames::useTag))
+            continue;
+        const RenderStyle& style = renderer->style();
+        if (style.display() == DisplayType::None || style.visibility() != Visibility::Visible)
+            continue;
+        clipContentRepaintRect.unite(renderer->localToParentTransform().mapRect(renderer->repaintRectInLocalCoordinates(repaintRectCalculation)));
```

Comment:
There is no localToParentTransform in LBSE.

Reply:
Good point, removed.